rotkehlchen/chain/ethereum/defi/protocols.py

```diff
@@ -0,0 +1,113 @@
+from typing import NamedTuple
```

Comment:
Add a docstring to this file at the very beginning explaining what this file and its mapping is for